src/structure/grid.ts

```diff
@@ -727,6 +710,30 @@ export class ViewersGrid {
 
         return newGUID;
     }
+
+    /**
+     * Function to return *optimal* arrangement of n widgets.
+     * Defaults to a 4 x ? grid. Hardcoded for simplicity.
```

Comment:
```suggestion
     * Function to return *optimal* arrangement of n widgets.
```

It is no longer using a 4x? grid, so let's update this